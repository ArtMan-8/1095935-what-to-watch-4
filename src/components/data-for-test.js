export const moviePoster = {
  title: `Seven Years in Tibet`,
  genre: `Drame`,
  year: `1997`,
};

export const films = [{
  id: `1184`,
  title: `We need to talk about Kevin`,
  poster: `img/we-need-to-talk-about-kevin.jpg`,
},
{
  id: `1134`,
  title: `What We Do in the Shadows`,
  poster: `img/what-we-do-in-the-shadows.jpg`,
},
{
  id: `1148`,
  title: `Revenant`,
  poster: `img/revenant.jpg`,
},
{
  id: `1194`,
  title: `Johnny English`,
  poster: `img/johnny-english.jpg`,
}];
