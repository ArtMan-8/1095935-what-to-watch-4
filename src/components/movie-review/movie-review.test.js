import * as React from 'react';
import renderer from 'react-test-renderer';
import MovieReview from './movie-review';
import {comments} from '../data-for-test';

describe(`MovieReview`, () => {
  it(`Render MovieReview`, () => {
    const tree = renderer.create(
        <MovieReview
          review={comments[0]}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
