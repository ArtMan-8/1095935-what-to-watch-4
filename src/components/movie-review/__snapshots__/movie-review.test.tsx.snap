// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieReview Render MovieReview 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    />
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      />
      <time
        className="review__date"
      />
    </footer>
  </blockquote>
  <div
    className="review__rating"
  />
</div>
`;
